.travis.yml

```diff
@@ -71,6 +71,9 @@ matrix:
       if: 'tag IS NOT present AND (type = pull_request OR branch in (master, staging, staging2, trying) OR repo != nervosnetwork/ckb)'
       os: linux
       install:
+        - rustup update stable
```

Comment:
Please setup a new job to check fmt/clippy with the latest toolchain and allow failures in this job.